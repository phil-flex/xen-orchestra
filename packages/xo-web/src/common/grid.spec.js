/* eslint-env jest */

import React from 'react'
import { forEach } from 'lodash'
import { shallow } from 'enzyme'

import * as grid from './grid'

forEach(grid, (Component, name) => {
  it(name, () => {
    expect(shallow(<Component />)).toMatchSnapshot()
  })
})
