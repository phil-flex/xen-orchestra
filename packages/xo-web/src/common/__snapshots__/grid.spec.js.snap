// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Col 1`] = `
<div
  className="col-xs-12"
/>
`;

exports[`Container 1`] = `
<div
  className="container-fluid"
/>
`;

exports[`Row 1`] = `
<div
  className=" row"
/>
`;
